brian2/monitors/spikemonitor.py

```diff
@@ -237,16 +237,18 @@ def values(self, var):
         Examples
         --------
         >>> from brian2 import *
-        >>> G = NeuronGroup(2, """dv/dt = 100*Hz : 1
+        >>> G = NeuronGroup(2, """dv/dt = 100.0001*Hz : 1
```

Comment:
This is really ugly because it's actually in the documentation. Is there no way around this?

Reply:
I'll have a look, we can certainly come up with a different kind of example.

Reply:
There are quite a few examples like this (not just this one).